<launch>
	<node name="fr_night" pkg="place_detection_isl"
		output="screen"
		type="place_detection_node" >
		<param name="tau_w" type="int" value="3" />
		<param name="tau_n" type="int" value="1" />
		<param name="tau_p" type="int" value="15" />
		<param name="tau_avgdiff" type="double" value="0.15" />
		<param name="tau_inv" type="double" value="20" />
		<param name="tau_inv2" type="double" value="0.0" />
		<param name="camera_topic" type="string" value="/axis/image_raw" />
		<param name="image_width" type="int" value="640" />
		<param name="image_height" type="int" value="480" />
		<param name="focal_length_pixels" type="int" value="466" />
		<param name="tau_val_mean" type="double" value="0.1" />
		<param name="tau_val_var" type="double" value="0.01" />
		<param name="sat_lower" type="int" value="10" />
		<param name="val_lower" type="int" value="10" />
		<!-- DEBUG MODE SHOULD BE ALWAYS CHECKED -->
		<param name="debug_mode" type="bool" value="true" />
		<param name="file_path" type="string" value="/home/berkanhoke/Datasets/Freiburg/VVGamma/Night/" />
		<param name="use_previous_memory" type="bool" value="false" />
		<param name="previous_memory_path" type="string" value="/home/berkanhoke/Runs/Original/Sunny2" />
	</node>
</launch>
